templates/checkout/_partials/cart-voucher.tpl

```diff
@@ -34,7 +34,7 @@
               <li class="cart-voucher__item row">
                 <span class="cart-voucher__name col">{$voucher.name}</span>
                 <div class="d-flex align-items-center justify-content-end col">
-                  <span class="me-2">{$voucher.reduction_formatted}</span>
+                  <span class="me-2 fw-bold">{$voucher.reduction_formatted}</span>
                     {if isset($voucher.code) && $voucher.code !== ''}
```

Comment:
We should change this in the core. There should be a property `deletable` or something.

Reply:
Yeah but not related to hummingbird